examples/offloader.js

```diff
@@ -31,53 +33,54 @@ if (fs.existsSync('polldata.json')) {
 
 client.logOn(logOnOptions);
 
-client.on('loggedOn', function() {
-	console.log("Logged into Steam");
+client.on('loggedOn', function () {
+	console.log('Logged into Steam');
 });
 
-client.on('webSession', function(sessionID, cookies) {
-	manager.setCookies(cookies, function(err) {
```

Comment:
@DoctorMcKay yeah, but there is the error checking in (almost?) all functions. And if there's not, I guess it would benefit us in the future when we'd want to implement the error checking.